### DISCUSSION TITLE
Examen certificacion

### DISCUSSION THREAD

**Discussion by MaferMtz** [Launch-X-Latam/MisionFrontEnd#108] [Q&A]
Hola, me gustaria saber si habra algun examen de certificacion o algo asi para este programa ? Algo asi como en la edicion de innovaccion virtual con la certificacion de Azure pero con alguna certificacion de FrontEnd ?

**Comment by doguedogue** [comment]
No, este programa no tiene certificación en fullstack. 

!SPOILER ALERT!
Sin embargo al final hay un hackathon para que pruebes tus habilidades!

**Reply by jenniferguerreroc** [reply]
Hola, entonces no tiene certificado de ningun tipo?

**Reply by FaraN1ght** [reply]
@jennifernicoleg Según dicen en las clases, probablemente darán un papelito de haber participado pero no certificación. Lo importante es el conocimiento y los proyectos que agregues a tu repositorio GitHub, los certificados no te los revisan en el ámbito laboral.

**Reply by gjordanra** [reply]
Estoy de acuerdo en que lo bueno es que haremos los proyectos y con eso vamos a practicar, pero los certificados también son importantes porque muchísimos  reclutadores y empresas se fijan en los conocimientos que tienes y los certificados son una forma de comprobar que los tienes. Pero si quieres un certificado te recomiendo sitios gratis para que puedas sacarlos, uno muy bueno es OpenBootcamp, incluso tienes oportunidad de tomar una ruta enfocada en desarrollo web, diferente enfoque de las lecciones

**Reply by doguedogue** [reply]
Al final tendrás un portafolio de distintos proyectos que has logrado hacer en tiempo record.

- Levantamientos de requerimientos
- Diseño UX/UI
- HTML
- CSS
- JavaScript / Java
- etc.

Eso es más que un certificado que dice que sabes hacer ya que está será una prueba de lo que hiciste.

**Comment by JosafatJimenezB** [comment]
Para el caso de launchx solo entregan una constancia de participación, si entran a las cloud skills ahí si les proporcionan un Boucher para la certificación az-900

**Reply by ayax-m** [reply]
Pero el AZ-900 no está relacionado con desarrollo web, la AZ-204 sería más apropiada.

**Reply by JosafatJimenezB** [reply]
Es la que te ofrece innovaccion, ofrece todo lo básico para aprender sobre su nube lo cual ayuda mucho a como desplegar diversos tipos de aplicaciones y servicios
